File: Web/Angular/src/app/models/chamado/registrarProgressoChamadoViewModel.ts
export class RegistrarProgressoChamadoViewModel {
  comentario?: string;
  idChamado?: number;
  percentual?: number;

  constructor(
    comentario?: string,
    idChamado?: number,
    percentual?: number,
 ){
    this.comentario = comentario;
    this.idChamado = idChamado;
    this.percentual = percentual;
  }
}

File: Context/Chamados/Application/ViewModels/Chamado/RegistrarProgressoChamadoViewModel.cs
namespace Chamados.Application.ViewModels.Chamado
{
    /// <summary>
    ///
    /// </summary>
    public class RegistrarProgressoChamadoViewModel
    {
        /// <summary>
        ///
        /// </summary>
        public RegistrarProgressoChamadoViewModel()
        {
            Comentario = string.Empty;
        }

        /// <summary>
        ///
        /// </summary>
        public string Comentario { get; set; }

        /// <summary>
        ///
        /// </summary>
        public int IdChamado { get; set; }

        /// <summary>
        ///
        /// </summary>
        public double Percentual { get; set; }
    }
}